### PR TITLE
Add inlay parameter name hints for call expr

### PR DESCRIPTION
**Pull request by imtsuki**
This patch adds Intellij-like parameter name hints for literal values in function calls.

Screenshot

Signed-off-by: imtsuki <me@qjx.app>

### CODE REVIEW

**Comment by imtsuki** [comment]
Oops, I forgot to update the test snapshot.

File: crates/ra_ide/src/display/function_signature.rs [hunk 1]
```diff
@@ -157,6 +162,22 @@ impl From<&'_ ast::FnDef> for FunctionSignature {
...
+            let mut res = vec![];
+            if let Some(param_list) = node.param_list() {
+                if let Some(self_param) = param_list.self_param() {
+                    res.push(self_param.syntax().text().to_string())
+                }
+
+                res.extend(
+                    param_list
+                        .params()
+                        .map(|param| param.pat().unwrap().syntax().text().to_string()),
```
Comment (matklad) [inline crates/ra_ide/src/display/function_signature.rs]: This `unwarp` can fire: all nodes in the AST might be incomplete

Comment (imtsuki) [reply crates/ra_ide/src/display/function_signature.rs]: You’re right. Maybe I should make `parameter_name_list` a `Vec<Option<String>>` here?

Comment (matklad) [reply crates/ra_ide/src/display/function_signature.rs]: I'd do just `.unwrap_or_default()` for simplicity

**Comment by imtsuki** [comment]
comment addressed

**Comment by matklad** [comment]
bors r+

Thanks!

**Pull request merged**